spec/fixtures/sample.js

```diff
@@ -10,4 +10,20 @@ var quicksort = function () {
   };
 
   return sort(Array.apply(this, arguments));
-};
\ No newline at end of file
+};
+
+foo({
+    atIndentTwo: true
```

Comment:
well, as I said, that won't be possible with the current single-line look-back logic. Then we'll need to leave all these indentation bugs in the code until you release the AST logic you mentioned or bite the bullet and search back to the beginning of the scope -- which I believe I saw @svanharmelen trying to argue for but failed.

Frankly, I feel that putting aesthetics in specific case over general correctness (that follows some logic) is not a good idea. I don't take node.js docs to be the ground truth of what is correct indentation. And I don't think your editor should only indent correctly when you are writing clean code. It should *help* you write clean code, not try to mislead you when you don't. I really don't see what's wrong with

```js
fs.readFile('/etc/passwd', (err, data) => {
    // indented two levels
});
```

But if that's what you want, then again, you will have to give up correct indentation in many other cases such as this one:

```js
const child = exec('cat *.js bad_file | wc -l',
  (error, stdout, stderr) => {
    console.log(`stdout: ${stdout}`);
    console.log(`stderr: ${stderr}`);
    if (error !== null) {
      console.log(`exec error: ${error}`);
    }
  });
  // this should be at indentation zero, not one!
```